ucs/chatchannel.cpp

```diff
@@ -78,7 +78,7 @@ ChatChannel *ChatChannelList::CreateChannel(
 {
 	uint8 max_perm_player_channels = RuleI(Chat, MaxPermanentPlayerChannels);
 
-	if (!database.CheckChannelNameFilter(name)) {
+	if (!database.CheckChannelNameFilter(name) && !RuleB(Chat, ChannelsIgnoreNameFilter)) {
```

Comment:
Would put rule first so we’re not doing an unnecessary query here.

Reply:
Good call. Fixing now